rllib/algorithms/algorithm.py

```diff
@@ -667,54 +667,44 @@ def setup(self, config: AlgorithmConfig) -> None:
             )
             self.config.off_policy_estimation_methods = ope_dict
 
-        # Create a set of env runner actors via a EnvRunnerGroup.
-        self.env_runner_group = EnvRunnerGroup(
```

Comment:
Oh wow, this is kind of cool. So offline algos won't have any `EnvRunnerGroup` anymore at all?!

If all tests pass with this, I'm totally fine with it :D But yeah, should work: The Learner now carries the main RLModule copy, so what was previously the task of the "local" EnvRunner is now not needed anymore.